lib/instrumentation/modules/mysql.js

```diff
@@ -98,59 +98,61 @@ function wrapQueryable (connection, objType, agent) {
 
   function wrapQuery (original) {
     return function wrappedQuery (sql, values, cb) {
-      var span = agent.startSpan(null, 'db', 'mysql', 'query')
-      var id = span && span.transaction.id
+      agent.logger.debug('intercepted call to mysql %s.query', objType)
+
+      var span = ins.createSpan(null, 'db', 'mysql', 'query')
+      if (!span) {
```

Comment:
+1 for bailing out early if the span didn't start -- there doesn't appear to be any significant code path that follows in the function for an undefined span.